### ISSUE TITLE
Behavioral difference from /source to /lib versions

### ISSUE THREAD

**Issue by jessevondoom** [substation-me/lodge#7]
The /lib library is a transpiled, minified version of the /source folder, generated by Babel. Looking at the testing site most functionality is working fine whether you use the /source or /lib versions of Lodge. There's one exception:

When you click one of the two embeds a menu comes up. Clicking the "say hello" link works, but the "change things up" link, which triggers the embeds to randomly shuffle background colors and phrases, does nothing. The event is relayed, but somehow not caught/addressed. 

(There is a third embed hosted off-site with an older version of Lodge. Known breaking changes were made so it's correctly not showing up. Will fix, but it's unrelated to this issue...) 

Unsure of the severity of this. Some next steps:

- Try with mix-and-match scripts using /source and /lib, see if they make a difference
- Check the program flow on the shuffle script to see where it fails, expanding the messaging for ?debug=true if needed
- Look into Babel options to see if we're just dropping an object name mistakenly or if there's a workaround

More to come.

**Comment by jessevondoom** [comment]
Update:

The startshuffle event relays correctly with either the /source or /lib versions of lodge. When the index page is using /source it works correctly and targets all three embeds with the doshuffle event as defined in the body of the page. When index uses /lib it never recognizes startshuffle OR fires the targeted doshuffle events. Here's a look at correct execution in the debug console:

![Screen Shot 2020-07-15 at 1 52 07 PM](https://user-images.githubusercontent.com/368640/87578443-6afa1f80-c6a2-11ea-87de-a2a3767b7f78.png)

So narrowing things down, this looks to be an issue either with the ready event (which wraps the startshuffle listener) or with the custom even listeners in general.

**Comment by jessevondoom** [comment]
Okay so in the inline script on index, the ready event is caught (verified by a console.log) and the doshuffle event can be fired/targeted correctly (verified by a 5 second delay call to fire the event inside the ready block) — so our issue revolves around the startshuffle event never firing/getting caught. 

I'm inclined to think the transpiling may have exposed a bug in the error relaying rather than truly broken it. Inspecting...

**Comment by jessevondoom** [comment]
Was right that the transpiling exposed an actual bug, but also decided the transpiling doesn't really buy us much, so went with a minify-only scheme. 

Nutshell on what was happening: 
The "lodgemessage" value of an incoming message was always set to true, no matter the source. so lodge was catching EVERYTHING and relaying it like a dumb jerk. The transpiling put us back to checking sources (correctly) and we were no longer authorizing correctly when we targeted specific embeds. Fixed that up and started looking into hardening message detection a little more. So yay!

But the transpiling added a ton of weight, and ultimately broke on a few other issues. It's just not worth it. There's nothing wild happening in there and most modern browsers can access all of lodge. So...fuck it?
